pkg/artifactory/resource_artifactory_local_repository_test.go

```diff
@@ -661,3 +662,56 @@ func TestAccLocalAllRepoTypes(t *testing.T) {
 		})
 	}
 }
+
+//Usage of the function is strictly restricted to Test Cases
+func getValidRandomDefaultRepoLayoutRef() string {
```

Comment:
There is already a function that does this : `randSelect` - util.go:126

Reply:
Improved.
```
func getValidRandomDefaultRepoLayoutRef() string {
	return randSelect("simple-default", "bower-default", "composer-default", "conan-default", "go-default", "maven-2-default", "ivy-default", "npm-default", "nuget-default", "puppet-default", "sbt-default").(string)
}
```
moved the test specific code to util_test.go